### PR TITLE
Fixed a few grammatical mistakes.

### PR DESCRIPTION
**Pull request by thesohelshaikh**
All changes were accepted and tested using app.grammarly.com.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,8 +68,8 @@ Courses in Core CS should be taken linearly if possible, but since a perfectly l
 each class's prerequisites is specified so that you can design a logical but non-linear progression
 based on the class schedules and your own life plans.
 
-**Cost**. All or nearly all course material prior to Pro CS is available for free,
-however some courses may charge money for assignments/tests/projects to be graded.
+**Cost**. All or nearly all course material prior to Pro CS is available for free.
+However, some courses may charge money for assignments/tests/projects to be graded.
```
Comment (thesohelshaikh) [inline README.md]: Missing comma after introductory phrase

File: README.md [hunk 2]
```diff
@@ -432,7 +432,7 @@ via a pull request and use our [community](#community) channels to announce it t
 
 Your peers and mentors from OSSU will then informally evaluate your project.
 You will not be "graded" in the traditional sense — everyone has their own measurements for what they consider a success.
-The purpose of the evaluation is to act as your first announcement to the world that you are a computer scientist,
+The purpose of the evaluation is to act as your first announcement to the world that you are a computer scientist
```
Comment (thesohelshaikh) [inline README.md]: Comma not required here.

**Comment by hanjiexi** [comment]
Thank you! It seems I am in need of a refresher on proper grammar...

**Comment by theumairahmed** [comment]
Guys please, how the fuck do I unsubscribe to this? Every single comment on
this forum pops up as an email on my phone. It’s a mess!

**Comment by hanjiexi** [comment]
It's possible you may have "watched" this repository which would notify you of every conversation. Watching isn't terribly useful unless you are a maintainer or a significant stakeholder; you may want to Star the repository instead if you just want to "bookmark" it so to speak.

Go to the top right of the screen and press "Unwatch", to the left of "Star".

**Comment by lhaither** [comment]
At bottom there is a MUTE THREAD

Sent from Mail for Windows 10

**Pull request merged**